glide.yaml

```diff
@@ -3,7 +3,7 @@ ignore:
 - go/doc
 import:
 - package: github.com/uber-go/tally
-  version: ^v2.1.0
+  version: ^v3
```

Comment:
you need to run `glide up` after changing it, and i think you can do just `^3`

Reply:
I ran `glide up`, but github hides `glide.lock` changes by default.
Changing `^v3` to `^3`.